baby-liminal-extension/src/error_codes.rs

```diff
@@ -0,0 +1,25 @@
+//! Error codes for the baby-liminal-extension pallet.
```

Comment:
nit: arent those a status code? Since success variant is amoung them

Reply:
renamed, thanks